src/webgl/p5.RendererGL.js

```diff
@@ -1048,4 +1049,12 @@ p5.RendererGL.prototype._vToNArray = function(arr) {
   );
 };
 
+/**
+ * ensures that p5 is using a 3d renderer. throws an error if not.
+ */
+p5.prototype._assert3d = function() {
+  if (!this._renderer.isP3D)
+    throw 'Not supported in P2D mode. Please use webgl mode';
```

Comment:
Maybe we can take this as an opportunity to update this message to offer new users a hint for how to fix the issue. Here are some suggestions, trying to build on what I see in the friendly error system:

* (method name) is only supported in WEBGL mode. If you'd like to use 3D graphics and Web GL, see https://p5js.org/examples/form-3d-primitives.html for more information.
* (method name) is not supported in 2D mode. Please create the canvas using WEBGL mode, see https://p5js.org/reference/#/p5/createCanvas for more information.

I think we could probably update the createCanvas docs to have a 3D example or we could have a more appropriate/simpler 3D sketch in the examples page, but that can wait until a future patch.